lib/proofer/applicant.rb

```diff
@@ -1,5 +1,6 @@
 module Proofer
   class Applicant
+    attr_accessor :uuid
```

Comment:
think this might be a good use for a comment, then? If I came into this file without that context I might be kind of confused